src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -49,6 +49,11 @@ public IRenderPackageFactory RenderPackageFactory
             get { return dynamoViewModel.RenderPackageFactoryViewModel.Factory; }
         }
 
+        public PackageManagerClientViewModel PackageManagerClientViewModel
```

Comment:
I don't think we should do this either 😉  - sorry for not being able to respond earlier - 
If there's no better way, than okay, but lets think more generally, are we going to do this for every view Model in Dynamo? It exposes too much of the application IMO and doesn't help people find what they need.

This seems like a very specific large surface to expose, and it exposes quite a lot of APIs ... 
What about adding just the apis you need (publish, download etc?) or event an event to request those?

Reply:
another way to think about it is to follow the example set out by the rest of the extension API - everything is exposed is an interface that describes a set of small apis that we consider required or useful, then you expose the object, but only as that interface, that way you control what apis we support.

Reply:
look at `IworkspaceModel` as an example, the extensions interfaces don't expose `WorkspaceModel` but give users an `IworkspaceModel` - they can cast this if they choose, but API users are making a decision to trade some safety for flexibility, and that is more clear.

Reply:
This makes sense. All we need to surface right now is "DownloadPackage". Let's discuss briefly tomorrow.